CONTRIBUTING.md

```diff
@@ -24,22 +24,22 @@ Retarget the solution to v141:
 
 
 ## Using RNTester
-RNTester is a React Native Windows App that demonstrates the implemented views and modules of React Native Windows.
+RNTester is a React Native Windows app that demonstrates the implemented views and modules of React Native Windows.
 
-You can use it to test your changes to react-native-windows by making sure your changes haven't broken the views and modules.
+You can use it to test your changes to React Native Windows by making sure your changes haven't broken the views and modules.
```

Comment:
Don't need to fix it here, but we have inconsistencies throughout the documentation of React Native Windows vs. react-native-windows. Honestly, I think I use both when writing about the project, so it probably doesn't matter.

Reply:
yeah I kinda view it as React Native Windows referring to the project as a whole, and react-native-windows as referring to the code. idk where I got that or if other people are using it like that.